OWNERS_ALIASES

```diff
@@ -221,3 +221,9 @@ aliases:
     - mfilocha
     - nvtkaszpir
     - kpucynski
+  sig-docs-ar-owners: # Admins for Arabic content
+    - mohamedsgap
```

Comment:
@mohamedsgap Normally it’s good to at least have a 2nd owner/approver on board as well. Otherwise an English speaking person always needs to be involved. Including a 2nd approver gives the Arabic localization team the ability to approve content themselves without intervention of English content approvers.

BTW thanks for starting this localization :tada: